From ff2877bc2d16dabb4ab6bd3a23c352c5b19215d2 Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Thu, 19 Jun 2014 10:44:44 -0400
Subject: [PATCH 042/118] sparsify: use Filename.temp_dir_name instead of
 get_temp_dir_name

Filename.get_temp_dir_name is new in ocaml >= 4.
---
 sparsify/sparsify.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sparsify/sparsify.ml b/sparsify/sparsify.ml
index 869dd09..fa5dd28 100644
--- a/sparsify/sparsify.ml
+++ b/sparsify/sparsify.ml
@@ -245,7 +245,7 @@ let () =
       virtual_size (human_size virtual_size)
 
 (* Check there is enough space in $TMPDIR. *)
-let tmpdir = Filename.get_temp_dir_name ()
+let tmpdir = Filename.temp_dir_name
 
 let () =
   let print_warning () =
-- 
2.7.4

